blog/models.py

```diff
@@ -1,15 +1,28 @@
 from django.db import models
+#from django.conf import settings
+#from django.utils import timezone
 
 # Create your models here.
 
+STATUS = (
+    (0,"Draft"),
```

Comment:
pep8 / flake, space after comma (try to run flake and/or balck upon commit) - the undone demo yesterday ;) 

https://docs.google.com/presentation/d/1O_86R0CWiruh0VQs_zOtFyx6JBZV3o5Fx4Ntcx3D7EM/edit#slide=id.geab85bd99d_0_0